source/reference/logs.txt

```diff
@@ -0,0 +1,54 @@
+.. _mdb-shell-logs:
+
+==================
+MongoDB Shell Logs
+==================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/admonitions/fact-mdb-shell-beta.rst
+
+|mdb-shell| stores logs for each session in `ndjson
+<http://ndjson.org/>`__ format using `pino
+<https://github.com/pinojs/pino>`__. With a Session ID, you can view or
+tail the logs for that |mdb-shell| session.
+
+View |mdb-shell| Logs
+---------------------
+
+.. include:: /includes/steps/view-logs.rst
+
+View |mdb-shell| Command History
+--------------------------------
+
+|mdb-shell| saves a history of all commands you've run across sessions.
+When a new command is issued, it is added to the top of the history
+file.
+
+.. tabs::
```

Comment:
Do tabs add much here? I understand wanting to be consistent with the previous section, but I'm wondering if just a table with a row for macOS / Linux and a row for Windows might be cleaner. I think either approach could work, but just something to think about.

Reply:
updated. I think the table is cleaner.